pkg/expapi/v1/defaults.go

```diff
@@ -16,4 +16,16 @@ limitations under the License.
 
 package v1
 
-func addDefaultingFuncs() {}
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func addDefaultingFuncs() {
+	api.Scheme.AddDefaultingFuncs(
+		func(obj *Scaler) {
+			if obj.Spec.Replicas == nil {
+				obj.Spec.Replicas = new(int)
```

Comment:
Why do you need a default value? It should just be populated from the underlying resource (ReplicationController in this case).